appinfo/info.xml

```diff
@@ -18,8 +18,8 @@
 	<screenshot>https://raw.githubusercontent.com/ChristophWurst/twofactor_gateway/3f895bc997c1a3a38c9bae16ccae8bd499af864e/screenshots/settings.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/ChristophWurst/twofactor_gateway/3f895bc997c1a3a38c9bae16ccae8bd499af864e/screenshots/challenge.png</screenshot>
 	<dependencies>
-		<php min-version="7.2" max-version="7.4"/>
-		<nextcloud min-version="18" max-version="20"/>
+		<php min-version="7.2" max-version="8.0"/>
+		<nextcloud min-version="18" max-version="21"/>
```

Comment:
please drop Nextcloud 18+19, only 20 is still supported. and then update the matrix at https://github.com/nextcloud/twofactor_gateway/blob/b1b7d805fd26fd9170993b8029cd2439e5274fef/.travis.yml#L65-L81 so tests are ran against 20 and 21. You may actually also add 22 as max version, which corresponds to the current `master` branch of Nextcloud.

Reply:
I have not yet used travis. I would remove line 2 from the travis file, because not all PHP version to check are available in 
 `trusty`, would add php 8.0 in the list of `php` and set the following as matrix:

```yaml
matrix:
  include:
    # Min PHP-Version allowed for NC 20
    - php: 7.2
      dist: trusty
      env: "DB=sqlite SERVER_BRANCH=stable20"
    # Min PHP-Version allowed for NC 21
    - php: 7.3
      dist: trusty
      env: "DB=sqlite SERVER_BRANCH=stable21"
    - php: 7.4
      dist: trusty
      env: "DB=sqlite SERVER_BRANCH=stable21"
    # Max PHP-Version allowed for NC 21 (n/a on trusty)
    - php: 8.0
      dist: xenial
      env: "DB=sqlite SERVER_BRANCH=stable21 PHP_COVERAGE=TRUE"
    - php: nightly
      dist: xenial
      env: "PATCH_VERSION_CHECK=TRUE"
```

I was also not able to find a hint on min PHP Version for NC22. nextcloud/documentation does not contain a branch for 22 yet, nor was I able to find a hint in the code after a quick review.

Reply:
22 will support 7.3 to 8.0. Ref https://github.com/nextcloud/server/blob/master/lib/versioncheck.php

Reply:
Oh. I did not see your comment. 🙄 Will check today evening.